pkg/csi_driver/node.go

```diff
@@ -276,6 +282,26 @@ func (s *nodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublish
 	return &csi.NodePublishVolumeResponse{}, nil
 }
 
+func removeDisallowedMountOptions(fuseMountOptions []string, disallowedFlags map[string]bool) []string {
+	fuseMountOptionsStripped := []string{}
```

Comment:
Return early without creating extra slice  / parsing through old slice for common case (no disallowed allowed)]
```
	if len(disallowedFlags) == 0 {
		return fuseMountOptions
	}
```